models/users/snowplow_users.sql

```diff
@@ -1,7 +1,7 @@
 
 {{
     config(
-        materialized='incremental',
+        materialized='table',
```

Comment:
This is suspect -> you rebuild this entire table per run?

Reply:
🙈 fixed